SourceryTests/Parsing/TemplateAnnotationsParserSpec.swift

```diff
@@ -46,7 +46,7 @@ class TemplateAnnotationsParserSpec: QuickSpec {
                         "    var something: Int\n" +
                         "    // sourcery:end\n"
 
-                let result = TemplateAnnotationsParser.parseAnnotations("inline", contents: source)
+                let result = TemplateAnnotationsParser.parseAnnotations("inline", contents: source, aggregate: false, forceParse: [])
```

Comment:
We should have some tests that validate expectations when this attribute is set to something.

Reply:
> We should have some tests that validate expectations when this attribute is set to something.

Test added.